hooks/multi_publish/nuke/secondary_publish.py

```diff
@@ -656,8 +657,11 @@ def _update_flame_clip(self, clip_path, write_node, sg_publish, progress_cb):
         # and lastly plug in the values
         render_path_fields["SEQ"] = format_str % (min_frame, max_frame) 
         
-        # contruct the final path             
-        publish_path_flame = publish_template.apply_fields(render_path_fields)
+        # contruct the final path - because flame doesn't have any windows support and 
+        # because the "hub" platform is always linux (with potential flame assist and flare
+        # satellite setups on macosx), request that the paths are written out on linux form
+        # regardless of the operating system currently running.
+        publish_path_flame = publish_template.apply_fields(render_path_fields, TemplatePath.PLATFORM_LINUX)
```

Comment:
Gut feeling - having to import TemplatePath just to bring in the constant PLATFORM_LINUX feels messy - could this instead just use the same constant/platform string as sys.platform would return on Linux (e.g. linux2)?